infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -292,7 +287,9 @@ public synchronized void handleMessageLayoutPropose(CorfuPayloadMsg<LayoutPropos
 
         long serverEpoch = getServerEpoch();
 
-        if (msg.getPayload().getEpoch() != serverEpoch) {
+        // If both epoch numbers in the msg aren't equal, then the request is malformed.
+        if (msg.getPayload().getEpoch() != msg.getPayload().getLayout().getEpoch() ||
```

Comment:
That sounds reasonable, but ... I don't see an existing CorfuMsgType that would allow the server to signal that exception to the client.  Options to proceed include:

1. Drop these malformed request checks (in both places) from this PR.
2. Drop these malformed request checks (in both places) from this PR, create new bug to add the checks + new message type + throw on client side.
3. Keep malformed checks as-is in this PR, create new bug to add new message type + throw on client side.

Thoughts?